packages/venia-ui/lib/components/SearchPage/searchPage.css

```diff
@@ -23,12 +23,21 @@
 }
 
 .filterButton {
-    padding: 0.5rem;
     margin-left: 0.5rem;
-    margin-right: 0.5rem;
-    width: 9rem;
+    display: inline-block;
+    padding: 0.75rem 2rem;
     border: 1px solid black;
-    border-radius: 100px;
+    border-radius: 3px;
     color: black;
     outline: none;
 }
+
+.sortContainer {
+    font-size: 0.875rem;
+    font-style: italic;
```

Comment:
Avoid using `italic`. We don't load that font style (or want to), so the browser has to emulate it, which is no good. To reduce the prominence of text, use a lighter color or decrease the size.

Reply:
Blindly followed the mock, good to know. Removed in 5cf9e9a.